client/src/components/Lobby.svelte

```diff
@@ -96,11 +97,11 @@
     {/if}
   </div>
   <SelectAvatar {lobby} {lobbyCode} on:change={(event) => onAvatarSelect(event.detail.value)} />
-  {#if $user?.uid === lobby.uids[0]}
+  {#if $user?.uid === lobby.host}
     <div class="actions">
-      <Button on:click={start} disabled={lobby.players.length < minPlayers || waiting}>
-        <Label>Start Game</Label>
-      </Button>
+      <Button on:click={() => start()} disabled={playersLength < minPlayers || waiting}
```

Comment:
That's only changing due to a merge from dev. It's been like that on since shunsuke did the once button fix. That's currently how it is on dev/prod as well